test/specs/token-assert.js

```diff
@@ -77,70 +79,174 @@ describe('modules/token-assert', function() {
+                    token: tokens[0],
+                    nextToken: tokens[1],
+                    exactly: 2
+                });
+
+                assert(onError.calledOnce);
+
+                var error = onError.getCall(0).args[0];
+                assert.equal(error.message, '2 spaces required between x and =');
+                assert.equal(error.line, 1);
+                assert.equal(error.column, 1);
             });
 
-            assert(!onError.called);
-        });
+            it('should not trigger error on newline between tokens', function() {
+                var file = createJsFile('x\n=y;');
 
-        it('should not trigger error on valid space count between tokens', function() {
-            var file = createJsFile('x   =   y;');
+                var tokenAssert = new TokenAssert(file);
+                var onError = sinon.spy();
+                tokenAssert.on('error', onError);
 
-            var tokenAssert = new TokenAssert(file);
-            var onError = sinon.spy();
-            tokenAssert.on('error', onError);
+                var tokens = file.getTokens();
+                tokenAssert.spacesBetween({
+                    token: tokens[0],
+                    nextToken: tokens[1],
+                    exactly: 2
+                });
 
-            var tokens = file.getTokens();
-            tokenAssert.whitespaceBetween({
-                token: tokens[0],
-                nextToken: tokens[1],
-                spaces: 3
+                assert(onError.notCalled);
+            });
+
+            it('should not trigger error on valid space count between tokens', function() {
+                var file = createJsFile('x   =   y;');
+
+                var tokenAssert = new TokenAssert(file);
+                var onError = sinon.spy();
+                tokenAssert.on('error', onError);
+
+                var tokens = file.getTokens();
+                tokenAssert.spacesBetween({
+                    token: tokens[0],
+                    nextToken: tokens[1],
+                    exactly: 3
+                });
+
+                assert(onError.notCalled);
+            });
+
+            it('should accept message for invalid space count between tokens', function() {
+                var file = createJsFile('x   =   y;');
+
+                var tokenAssert = new TokenAssert(file);
+                var onError = sinon.spy();
+                tokenAssert.on('error', onError);
+
+                var tokens = file.getTokens();
+                tokenAssert.spacesBetween({
+                    token: tokens[0],
+                    nextToken: tokens[1],
+                    exactly: 2,
+                    message: 'Custom message'
+                });
+
+                assert.equal(onError.getCall(0).args[0].message, 'Custom message');
             });
 
-            assert(!onError.called);
+            it('should error, but not fix, when a comment exists between the two tokens', function() {
```

Comment:
Should there be a test for `should error and fix whitespace between tokens without a comment`